Mathlib/LinearAlgebra/CliffordAlgebra/SpinGroup.lean

```diff
@@ -0,0 +1,419 @@
+/-
+Copyright (c) 2022 Jiale Miao. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jiale Miao, Utensil Song, Eric Wieser
+-/
+import Mathlib.GroupTheory.GroupAction.ConjAct
+import Mathlib.Algebra.Star.Unitary
+import Mathlib.LinearAlgebra.CliffordAlgebra.Star
+import Mathlib.LinearAlgebra.CliffordAlgebra.Even
+import Mathlib.LinearAlgebra.CliffordAlgebra.Inversion
+
+/-!
+# The Pin group and the Spin group
+
+In this file we define `lipschitzGroup`, `pinGroup` and `spinGroup` and show they form a group.
+
+## Main definitions
+
+* `lipschitzGroup`: the Lipschitz group with a quadratic form.
+* `pinGroup`: the Pin group defined as the infimum of `lipschitzGroup` and `unitary`.
+* `spinGroup`: the Spin group defined as the infimum of `pinGroup` and `CliffordAlgebra.even`.
+
+## Implementation Notes
+
+The definition of the Lipschitz group
+$\\{ x \in \mathop{\mathcal{C}\ell} | x \text{ is invertible and } x v x^{-1} ∈ V \\}$ is given by:
```

Comment:
I'd prefer that we PR the "broken" $\LaTeX$, so that when doc-gen is eventually fixed it ends up less broken rather than more broken:
```suggestion
$\{ x \in \mathop{\mathcal{C}\ell} | x \text{ is invertible and } x v x^{-1} ∈ V \}$ is given by:
```

Reply:
I'm OK with either, but I'm not very optimistic about when it would be fixed.